protocol/messenger_emoji_reactions.go

```diff
@@ -20,12 +42,28 @@ func (m *Messenger) SendEmojiReaction(ctx context.Context, chatID, messageID str
 	}
 	clock, _ := chat.NextClockAndTimestamp(m.getTimesource())
 
+	if emoji == "" {
+		emoji = ConvertEmojiIDToString(emojiID)
+		if emoji == "" {
+			return nil, errors.New("invalid emojiID")
+		}
+	}
+
+	// Validate that the emoji is valid
+	// There is no foolproof way to validate emojis, but this regex should cover all standard emojis
+	// it will also allow some non-emoji unicode characters, but that's not a big issue
+	emojiRegex := regexp.MustCompile("(?:\u00A9|\u00AE|[\u2000-\u3300]|[\U0001F000-\U0001FBFF])")
```

Comment:
Please extract this `regex` as a global variable in the beginning of the file. So that we don't compile it every time the we send an emoji. 

Perhaps also makes sense to validate incoming emojis?